plugins-builtin/src/main/scala/za/co/absa/enceladus/plugins/builtin/errorinfo/mq/kafka/KafkaErrorInfoPlugin.scala

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.plugins.builtin.errorinfo.mq.kafka
+
+import com.typesafe.config.Config
+import org.apache.avro.{Schema => AvroSchema}
+import org.apache.commons.io.IOUtils
+import org.apache.spark.sql.avro.SchemaConverters
+import org.apache.spark.sql.types.StructType
+import za.co.absa.abris.avro.read.confluent.SchemaManager
+import za.co.absa.enceladus.plugins.api.postprocessor.PostProcessorFactory
+import za.co.absa.enceladus.plugins.builtin.common.mq.kafka.KafkaConnectionParams
+import za.co.absa.enceladus.plugins.builtin.errorinfo.mq.ErrorInfoSenderPlugin
+
+/**
+ * Implementation of a plugin factory that creates the above plugin based on configuration passed from
+ * Enceladus Spark application.
+ */
+object KafkaErrorInfoPlugin extends PostProcessorFactory {
```

Comment:
A couple of optional suggestions about naming:
1. We associate Info with _INFO files, so in this case this word might be redundant. Consider `KafkaErrorPlugin`
2. You have `KafkaErrorInfoPlugin` and `ErrorInfoSenderPlugin`. But one of these is a factory for the other. Maybe name one of them `KafkaErrorSenderPlugin` (factory) and the other `KafkaErrorSenderPluginImpl` (the actual plugin)

Please, feel free to disagree, I'm not quite sure myself. The confusion is caused by the plugin API that requires to specify the factory object in the configuration. But for simplicity for the users, we don't call them factories since
```
standardization.plugin.postprocessor.1=SomePlugin
```
looks more natural than
```
standardization.plugin.postprocessor.1=SomePluginFactory
```

Reply:
Good idea on the naming - I went with #2, so now it is `KafkaErrorSenderPlugin` & `KafkaErrorSenderPluginImpl`. Also, got rid of the "ErrorInfo" wording, too.